packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.1]
+### Changed
+- Relax dependency on `@metamask/base-controller`, `@metamask/controller-utils`, and `@metamask/network-controller` (use `^` instead of `~`) ([#998](https://github.com/MetaMask/controllers/pull/998))
+- Align with changes to `@metamask/network-controller`'s `NetworkState` type (`provider` -> `providerConfig`) ([#995](https://github.com/MetaMask/controllers/pull/995))
```

Comment:
This one is a breaking change because the `getNetworkState` constructor parameter is now expected to return state with a `providerConfig` property.

Reply:
Updated in [a24e1cc](https://github.com/MetaMask/controllers/pull/999/commits/a24e1ccf93fe97dd79dcba59200b3a43bdd4bf95).